LayoutTests/svg/text/textPathBoundsBug.svg

```diff
@@ -7,6 +7,11 @@
 
 <script>
 if (window.testRunner)
-    window.testRunner.dumpSelectionRect();
```

Comment:
I don't understand those changes, can you elaborate?

Reply:
We had `window.testRunner.dumpSelectionRect` but no selection, so I did changes to test without using `execCommand` (which is not allowed in non-HTML) but using `Selection API` to do 'selection' so we can dump them properly. @nikolaszimmermann